lineman/app/components/mini_proposal_pie_chart/mini_proposal_pie_chart_controller.coffee

```diff
@@ -16,12 +16,10 @@ angular.module('loomioApp').controller 'MiniProposalPieChartController', ($scope
   refreshPieChartData = ->
     return unless $scope.proposal
     counts = $scope.proposal.voteCounts
-    hasAnyVotes = counts.yes + counts.abstain + counts.no + counts.block > 0
     $scope.pieChartData[0].value = counts.yes
     $scope.pieChartData[1].value = counts.abstain
     $scope.pieChartData[2].value = counts.no
     $scope.pieChartData[3].value = counts.block
-    $scope.pieChartData[4].value = if hasAnyVotes then 0 else 1
+    $scope.pieChartData[4].value = if _.some [counts.yes, counts.abstain, counts.no, counts.block] then 0 else 1
```

Comment:
Yeah, there's likely a fancy elegant `_.map` solution here with `$scope.proposal.voteCounts -> $scope.pieChartData` but I wasn't up to puzzling it out at the time.